### ISSUE TITLE
npm install

### ISSUE THREAD

**Issue by ishaqkfoodhub** [stevekinney/redux-incident-counter#3]
nom install, keep getting errors with trying to install

**Comment by tprince232** [comment]
I had errors with `npm install`, but I saw the `yarn.lock` file so I used a `yarn install`.  Still had errors.  Looked like the error log was showing issues with the `node-sass` installation.  Troubleshooting showed me that I might have needed to specify where python was located, so I added an `.npmrc` file at the project root and added the contents `python = "/usr/bin/python2.7"`, where my python2 was installed on my Mac.  Still had an install issue, so I downgraded my node version to `v12.22.10`.  Then my `yarn install` completed successfully.

**Comment by dtauer** [comment]
Yes, we recommend using Node version `12.22.12` for this course.
